### PR TITLE
New version: OpslyCloud.opsly-cli version v0.0.1-alpha.17

### PR DESCRIPTION
**Pull request by stephenafamo**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/138276)

### CODE REVIEW

**Comment by stephenafamo** [comment]
@microsoft-github-policy-service agree company="Opsly Cloud LTD"

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2024-02-07T18:34:10.8028613Z ##[error] One or more errors occurred. (Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\o\OpslyCloud\0.0.1-alpha.17', expected 'manifests\o\OpslyCloud\opsly-cli\0.0.1-alpha.17') Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\o\OpslyCloud\0.0.1-alpha.17', expected 'manifests\o\OpslyCloud\opsly-cli\0.0.1-alpha.17'

(Automated response - build 708.)

**Comment by stephengillie** [comment]
Hi @

Can `\opsly-cli` be added to the file paths? Currently the path is:

`manifests\o\OpslyCloud\0.0.1-alpha.17`

But needs to be:

`manifests\o\OpslyCloud\opsly-cli\0.0.1-alpha.17`

**Comment by stephengillie** [comment]
Hi @stephenafamo,

The package didn't pass a Defender or similar security scan. This might be a false positive and we can rescan tomorrow.

(Automated response - build 720.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Installation failed with exit code -1978335187 2024-02-08 15:55:27.524 [FAIL] Installer failed security check. Url: https://github.com/Opsly/homebrew-cli/releases/download/v0.0.1-alpha.17/opsly_Windows_i386.exe Result: 0x80004005

(Automated response - build 720.)

**Comment by stephengillie** [comment]
Hi @stephenafamo,

The package didn't pass a Defender or similar security scan. This might be a false positive and we can rescan tomorrow.

(Automated response - build 720.)

**Comment by stephenafamo** [comment]
@stephengillie please how do we get the validation to run again?

**Comment by stephengillie** [comment]
> @ stephengillie please how do we get the validation to run again?

While my goal is to have this completely automated and retry every 18 hours, I haven't quite got that implemented yet.

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> No errors to post.

(Automated response - build 729.)

**Comment by stephengillie** [comment]
Hi @stephenafamo,

The package didn't pass a Defender or similar security scan. This might be a false positive and we can rescan tomorrow.

(Automated response - build 729.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Installation failed with exit code -1978335187 2024-02-12 16:49:23.460 [FAIL] Installer failed security check. Url: https://github.com/Opsly/homebrew-cli/releases/download/v0.0.1-alpha.17/opsly_Windows_i386.exe Result: 0x80004005

(Automated response - build 729.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> No errors to post.

(Automated response - build 745.)

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 5:24:49 PM: 
> 2024-02-14 17:24:48.975 [FAIL] Installer failed security check. Url: https://github.com/Opsly/homebrew-cli/releases/download/v0.0.1-alpha.17/opsly_Windows_x86_64.exe Result: 0x80004005

(Automated response - build 743.)

**Comment by stephengillie** [comment]
Hi @stephenafamo,

The package didn't pass a Defender or similar security scan. This might be a false positive and we can rescan tomorrow.

(Automated response - build 743.)

**Comment by stephengillie** [comment]
[Policy] Needs-Attention

**Comment by stephengillie** [comment]
[Policy] Validation-Defender-Error

**Comment by stephenafamo** [comment]
I've removed the offending binary @stephengillie

The checks seem to all be good now. Is there anything else I need to do?

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> No errors to post.

(Automated response - build 745.)

**Comment by stephengillie** [comment]
Manual Validation ended with:
> PS C:\Users\User\Desktop> opsly docker not found. Please install docker to run this tool https://docs.docker.com/engine/install/.

(Automated response - build 748.)

**Comment by stephengillie** [comment]
Should `Docker.DockerDesktop` be a dependency?

File: manifests/o/OpslyCloud/opsly-cli/0.0.1-alpha.17/OpslyCloud.opsly-cli.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# This file was generated by GoReleaser. DO NOT EDIT.
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+PackageIdentifier: OpslyCloud.opsly-cli
+PackageVersion: 0.0.1-alpha.17
+InstallerLocale: en-US
+InstallerType: portable
+Commands:
+  - opsly
+ReleaseDate: "2024-02-07"
```
Comment (stephengillie) [inline manifests/o/OpslyCloud/opsly-cli/0.0.1-alpha.17/OpslyCloud.opsly-cli.installer.yaml:9]:
```suggestion
ReleaseDate: "2024-02-07"
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Docker.DockerDesktop
```

Comment (stephengillie) [inline manifests/o/OpslyCloud/opsly-cli/0.0.1-alpha.17/OpslyCloud.opsly-cli.installer.yaml:9]:
```suggestion
ReleaseDate: "2024-02-07"
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Docker.DockerDesktop
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Should `Docker.DockerDesktop` be a dependency?

**Comment by stephenafamo** [comment]
Added the dependencies @stephengillie, but there is now another validation error. 
What needs to be fixed now?

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> No errors to post.

(Automated response - build 762.)

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 6:48:20 AM: 
> 2024-02-16 06:48:20.069 [FAIL] Installer failed security check. Url: https://github.com/Opsly/homebrew-cli/releases/download/v0.0.1-alpha.17/opsly_Windows_x86_64.exe Result: 0x80004005 6:48:21 AM: Detected 1 Defender (with signature version 1.405.20.0) errors: 6:48:21 AM: Trojan:Win32/Wacatac.B!ml

(Automated response - build 751.)

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 6:48:20 AM: 
> 2024-02-16 06:48:20.069 [FAIL] Installer failed security check. Url: https://github.com/Opsly/homebrew-cli/releases/download/v0.0.1-alpha.17/opsly_Windows_x86_64.exe Result: 0x80004005 6:48:21 AM: Detected 1 Defender (with signature version 1.405.20.0) errors: 6:48:21 AM: Trojan:Win32/Wacatac.B!ml

(Automated response - build 762.)

**Comment by stephengillie** [comment]
Hi @stephenafamo,

The package didn't pass a Defender or similar security scan. This might be a false positive and we can rescan tomorrow.

(Automated response - build 762.)

**Comment by stephenafamo** [comment]
@stephengillie the automated tests seem to pass now

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> No errors to post.

(Automated response - build 796.)

**Review by stephengillie — APPROVED** [review]

**Pull request merged**